tasks/yaml-lint.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
include this task from tekton hub via pipelinesAsCode annotation -> https://pipelinesascode.com/docs/guide/resolver/#remote-task-support

Reply:
Added below line in pull-request.yaml -
pipelinesascode.tekton.dev/task: "yaml-lint"

Reply:
This file can be now removed

Reply:
@Michkov would it make sense to move this task to a task folder under the pipeline as code directory Since this meant only for CI not to be distributed with hacbs pipelines?

Reply:
This task is not used now, `pipelinesascode.tekton.dev/task-2: "yaml-lint"` downloads it from tekton hub.

Reply:
But that's also option to move under .tekton/tasks and then point to it -> `pipelinesascode.tekton.dev/task-2: ".tekton/tasks/yaml-list.yaml"`, that's maybe better since we would have it under our control.

Reply:
@Michkov ah ok. I misread the conversation. So because we are using the upstream tekton task we are deleting this yaml-lint.yaml from tasks directory. 

But yeah maybe for future tekton tasks that aren't up on the tekton catalog, we should probably place tasks specific to the pipeline as code CI in a .tekton/tasks dir